src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -19,7 +19,9 @@
      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
    <% } else{ %>
      <a href="/login">Login</a>
+     <a href="/register">Register</a>
    <% } %>
+     <a href="/about.jsp">About</a>
```

Comment:
Nit: this could be indented two spaces less, so it lines up with the top-level features, not the indented if-else.

Note: this missing About link shows a good reason to think about using an include to only code this once!